fedimint-server/src/consensus/aleph_bft/keychain.rs

```diff
@@ -26,28 +28,15 @@ impl Keychain {
                 .private
                 .broadcast_secret_key
                 .keypair(secp256k1::SECP256K1),
+            tag: cfg.consensus.broadcast_public_keys.consensus_hash(),
```

Comment:
`consensus_hash` is more descriptive, `tag` is unclear to me. Might be just me, but just reporting my thoughts as I'm reading through it.